src/PowerShellEditorServices.Engine/Services/TextDocument/Handlers/CodeLensHandlers.cs

```diff
@@ -19,7 +19,7 @@ public class CodeLensHandlers : ICodeLensHandler, ICodeLensResolveHandler
         private readonly DocumentSelector _documentSelector = new DocumentSelector(
             new DocumentFilter()
             {
-                Pattern = "**/*.ps*1"
+                Language = "powershell"
```

Comment:
I'm planning on factoring this out into an abstract class or interface... just haven't gotten around to it.